src/psycopmlutils/loaders/raw/load_lab_results.py

```diff
@@ -1,90 +1,248 @@
-from typing import Optional
+from typing import Dict, Optional
 
 import pandas as pd
 
+from psycopmlutils.loaders.non_numerical_coercer import multiply_inequalities_in_df
 from psycopmlutils.loaders.raw.sql_load import sql_load
 from psycopmlutils.utils import data_loaders
 
 
 class LoadLabResults:
+    def concatenate_blood_samples(
+        blood_sample_ids: list,
+        n: Optional[int] = None,
+        values_to_load: str = "all",
+    ) -> pd.DataFrame:
+        """Concatenate multiple blood_sample_ids (typically NPU-codes) into one
+        column.
+
+        Args:
+            blood_sample_ids (list): List of blood_sample_id, typically an NPU-codes. # noqa: DAR102
```

Comment:
Seems `darglint` has a bug where the first argument to a method in a class gets a DAR102. Might be because these classes don't have an `__init__`. 

So not a mistake, but caused by a `darglint` bug :-)